test/apiv2/12-imagesMore.at

```diff
@@ -43,8 +43,29 @@ s1=$(jq -r .status <<<"${lines[1]}")
 like "$s1" "mytag: digest: sha256:[0-9a-f]\{64\} size: [0-9]\+" \
      "Push to local registry: second status line"
 
+# Push to local registry using the libpod endpoint with quiet=false...
+t POST "libpod/images/$IMAGE/tag?repo=localhost:$REGISTRY_PORT/myrepo&tag=mytag-1" 201
```

Comment:
and maybe use a different tag name than mytag-1, because it should more distinctly different from maytag so the people reading the logs or grep for the mytag are not confused by it.